### PR TITLE
Solve issue #193

### PR DESCRIPTION
**Pull request by egamxmx**
Solve #193 Add the category openSuse in reports Computer SO and Distribution Linux, without affect the actual operation, even if not find the sistem openSuse the reports work same

### CODE REVIEW

File: inc/inventory.class.php [hunk 1]
```diff
@@ -322,6 +322,7 @@ function computersByOS($config = []) {
       $sql_states   = self::getStateCondition('c.states_id');
       $oses = ['Windows' => 'Windows',
                'Linux'   => 'Linux|Ubuntu',
+               'openSUSE'  => 'openSUSE',
```
Comment (hvelarde) [inline inc/inventory.class.php:325]: seems to me this should be `'Linux' => 'Linux|Ubuntu|openSUSE',` instead.

Comment (egamxmx) [reply inc/inventory.class.php:325]: yes, your appreciation is correct, I fixed this line

**Comment by hvelarde** [comment]
you have to contact the maintainers of this repo in order to get this changes reviewed and merged, please.

**Comment by egamxmx** [comment]
@cedric-anne @orthagh can you help me with the merge of my change or tell me your feedback

**Review by orthagh — APPROVED** [review]

**Comment by orthagh** [comment]
Done, thanks for your contribution

**Comment by hvelarde** [comment]
@egamxmx now you have to follow up until we have a new version of the plugin; that's the way OSS works.

**Pull request merged**